intergov/processors/blockchain_router

```diff
@@ -0,0 +1 @@
+multichannel_router
```

Comment:
If this is a symlink, it won't work on Windows properly, better to avoid.

Reply:
I'll change anything except the Hamlet cloud devops so old directory is never referenced; after I fail to do so well please fix remaining places (when the PR is merged). The symlink is just for @roleyfoley to postpone devops changes till he has time.

Reply:
OK, understood, temporary stuff is fine.